lua/autorun/sh_cami.lua

```diff
@@ -372,11 +372,16 @@ local defaultAccessHandler = {["CAMI.PlayerHasAccess"] =
 
         if not priv then return callback(fallback, "Fallback.") end
 
-        callback(
+        local hasAccess =
             priv.MinAccess == "user" or
             priv.MinAccess == "admin" and actorPly:IsAdmin() or
             priv.MinAccess == "superadmin" and actorPly:IsSuperAdmin()
-            , "Fallback.")
+
+        if priv.HasAccess then
+            hasAccess = hasAccess and priv:HasAccess(actorPly, targetPly)
+        end
```

Comment:
Personally I'd prefer to keep all the conditionals in the if statement, but we have different coding styles
```suggestion
        if hasAccess and priv.HasAccess then
            hasAccess = priv:HasAccess(actorPly, targetPly)
        end
```